spec/uploaders/file_uploader_spec.rb

```diff
@@ -15,10 +15,6 @@
     end
   end
 
-  after do
```

Comment:
Wait until the review is complete, then squash? Because this is a framework change I think every commit should be checked that it passes all specs.